build/Microsoft.DotNet.Cli.DependencyVersions.props

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <CLI_NETSDK_Version>1.0.0-alpha-20170105-5</CLI_NETSDK_Version>
```

Comment:
no. I need to fix the ordering anyways and will do that while introducing CLI_NUGET_Version so we have a one-stop-shop for this PR in the future